README.md

```diff
@@ -208,6 +208,25 @@ client.on('message', (message) => {
 
 When you use the delete function, the giveaway data and the message of the giveaway are deleted. You cannot restore a giveaway once you have deleted it.
 
+### end a giveaway
+
+```js
+client.on('message', (message) => {
+    const args = message.content.slice(settings.prefix.length).trim().split(/ +/g);
+    const command = args.shift().toLowerCase();
+
+    if (command === 'end') {
+        let messageID = args[0];
```

Comment:
const

Reply:
What do u mean?

Reply:
let is unnecessary because you never reassign messageID, use const instead